core/src/avm2/globals/flash/events/ShaderEvent.as

```diff
@@ -12,7 +12,7 @@ package flash.events
     {
         public static const COMPLETE:String = "complete"; // Defines the value of the type property of a complete event object.
 
-        public var vector: Vector; // The Vector.<Number> object that was passed to the ShaderJob.start() method.
+        public var vector: Vector.<Number>; // The Vector.<Number> object that was passed to the ShaderJob.start() method.
```

Comment:
same in the constructor :)

Reply:
Bah, I made sure to do a search so I don't miss anything similar. And yet! In the very same file even! :facepalm: Thanks!